Assets/Scripts/Player/PlayerMovement.cs

```diff
@@ -6,7 +6,6 @@
 public class PlayerMovement : MonoBehaviour
 {
     public float Speed;
-

```

Comment:
Any particular reason why this line was deleted?

Reply:
It looked off

Reply:
I want to argue that it should stay as it separates the public and private variables. Unless we don't do this in other files?

Reply:
Thats find with me